basic-conditionals.tex

```diff
@@ -0,0 +1,400 @@
+    \PY{k+kt}{int} \PY{n}{zelja}\PY{p}{;}
+    \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Napisi 1, ce zelis juho, in 2, ce zelis sendvic.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{zelja}\PY{p}{)}\PY{p}{;}
+
+    \PY{k}{if} \PY{p}{(}\PY{n}{zelja} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{1}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Kaksno juho zelis? Napisi 1 za govejo in 2 za paradiznikovo.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{n}{zelja}\PY{p}{)}\PY{p}{;}
+        \PY{k}{if} \PY{p}{(}\PY{n}{zelja} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{1}\PY{p}{)} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Ena goveja juha. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Ena paradiznikova juha. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}}
+    \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Kaksen sendvic zelis? Napisi 1 za sunko, in 2 za vegi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{zelja}\PY{p}{)}\PY{p}{;}
+        \PY{k}{if} \PY{p}{(}\PY{n}{zelja} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{1}\PY{p}{)} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{En sendvic s sunko. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{En vegeterjanski sendvic. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{return} \PY{l+m+mi}{0}\PY{p}{;}
+\PY{p}{\PYZcb{}}
+\end{Verbatim}
+
+\begin{inout}
+Napisi 1, ce zelis juho, in 2, ce zelis sendvic.\\
+2\\
+Kaksen sendvic zelis? Napisi 1 za sunko, in 2 za vegi.\\
+2\\
+En vegeterjanski sendvic. Dober tek!
+\end{inout}
+\end{examples}
+
+\pagebreak
+Pozorni bodimo tudi na postavitev kode. Običajno kodo znotraj zavitih oklepajev \verb+if+ stavka pišemo tako, da je poravnana za štiri presledke desno od kode zunaj \verb+if+ stavka. V nekaterih programskih jezikih je taka poravnava obvezna, v C/C++ pa ne, vendar nezamaknjena koda že v zelo majhnih programih postane popolnoma nepregledna. Branje in popravljanje kode je veliko lažje, če del kode znotraj zavitih oklepajev zamaknemo za štiri presledke. Za to lahko uporabimo tudi tipko Tab, ki se na tipkovnici nahaja levo od tipke Q.
```

Comment:
Take vrstice je treba razlomit, ker je tezko podajat pripombe na vsebino tako dolgih vrstic in ker bo vsak diff v tej vrstici tezko pregledat in tudi sicer so dolge vrstice nehigienicne.
Tvoj editor ma zagotovo nekje funkcijo, ki nardi smiselne prelome vrstic besedila nekje okrog dolzine 80.

(to velja tud za vrstice ki so > 80 znakov in niso ta konkretna vrstica)